docs/pages/machine-workload-identity/mwi-use-cases/mwi-ci-cd.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: CI/CD with Machine & Workload Identity
+description: Configuring CI/CD with Machine & Workload Identity
+labels:
+ - conceptual
+ - mwi
+---
+
+Teleport Machine & Workload Identity eliminates the need for long-lived static secrets in CI/CD pipelines by issuing short-lived, automatically generated certificates at runtime.
+Teleport natively integrates with many CI/CD providers and deployment targets. Refer to the links at the bottom of this page for instructions on using Machine & Workload Identity with your preferred tools.
+
+## Choose your CI/CD platform
+
+<TileGrid
+  tiles={[
+    {
+      icon: <Icon name="azureDevops" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/azure-devops",
+      name: "Azure DevOps",
+    },
+    {
+      icon: <Icon name="bitbucket" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/bitbucket",
+      name: "BitBucket",
+    },
+    {
+      icon: <Icon name="circleci" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/circleci",
+      name: "CircleCI",
+    },
+    {
+      icon: <Icon name="gitlab" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/gitlab",
+      name: "GitLab CI",
+    },
+    {
+      icon: <Icon name="jenkins" size="xl" />,
+      to: "/machine-workload-identity/machine-id/deployment/jenkins",
+      name: "Jenkins",
+    }
+  ]}
+/>
+
+## Eliminate secrets from your CI/CD pipeline
+
+In a typical CI/CD setup, a pipeline such as one running in GitHub Actions on GitHub's infrastructure outside the corporate network builds a 
+container image, pushes it to a container registry, and updates an application running in a container orchestration platform (such as Kubernetes).
+
+This workflow commonly relies on sensitive credentials stored in the CI/CD system, such as:
+
+- Cloud provider access credentials (e.g., AWS, GCP, Azure)
+- Configuration files or tokens for deploying to orchestration platforms
+
+Best practices recommend rotating these secrets every six months. A platform team generates new credentials and shares them with a development team, 
+who must manually update the repository. Although the credential generation process emits a security log, teams often lack a reliable way to correlate 
+it with GitHub's audit logs to verify timely rotation. Sharing credentials via Slack, email, file drops, or password managers adds further risk. 
+Moreover, exposing the Kubernetes API to the internet introduces unnecessary surface area for attack.
+
+## Secure and auditable access with ephemeral credentials
+
+Teleport solves these problems by generating two ephemeral credentials at job start:
+
+- A [SPIFFE Verifiable Identity Document (SVID)](https://spiffe.io/spiffe/) used with [AWS IAM Roles Anywhere](https://docs.aws.amazon.com/rolesanywhere/latest/userguide/introduction.html) to authenticate with AWS ECR
```

Comment:
Hmmm, aren't the SPIFFE SVIDs more for the workload-workload case, and not so much for the CI/CD use case (which is the title of this document)?